config/relayer_1.json

```diff
@@ -36,6 +36,18 @@
       "gasLimit": 8000000,
       "maxGasPrice": 20000000000,
       "blockConfirmations": 10
+    },
+    {
+      "name": "polygon",
+      "type": "evm",
+      "id":3,
+      "endpoint":"wss://polygon-mumbai.g.alchemy.com/v2/NVK896meM89QkTJi7AmkHc8MaRa0z1PL",
```

Comment:
I feel like this shouldn't be public. Also, updating configs doesn't update dev environment so I don't think we need this update of configs anyway.

Reply:
It's created on some random account that is used only for api keys so it doesn't really matter. 
This is only to give a working playground, to spare people of deploying and setting up everything just to try it out.

Reply:
I think this can be merged as it is, as we already exposed these endpoints for other networks in configuration. 
I agree with Matija that there is no need to update this config with additional networks in the future, as this is only a sample configuration file.